PokeOneWeb/Frontend/pokeoneweb/angular.json

```diff
@@ -1,5 +1,8 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "cli": {
+    "analytics": false
```

Comment:
What does this do?

Reply:
when starting the angular application with "ng server" I was asked if I want to send analytic data ... I said no :)